TransformationSystem/Client/Transformation.py

```diff
@@ -132,6 +140,20 @@ def setBody(self, body):
           raise TypeError("Cannot encode %r, in json" % (val))
       return self.__setParam(json.dumps(body))
 
+  def setInputMetaQuery(self, query):
+    """Set the input meta query.
+
+    :param dict query: dictionary to use for input meta query
+    """
+    self.inputMetaQuery = query
+
+  def setOutputMetaQuery(self, query):
+    """Set the output meta query.
+
+    :param dict query: dictionary to use for output meta query
+    """
+    self.outputMetaQuery = query
```

Comment:
...do you really need these setter methods? And would they be invoked anywhere?

Reply:
They would be invoked by the users. (Or the ReplicationTransformation Utility)

Do you mean can I use properties instead Yes, but this is different than all the other parameters, so for consistency setters

Or do you mean can I use the same approach as for the other parameters which have an implicit setter?
I don't think so, because of the automatic call to setTransformationParameter or setAdditionalTransformationParameter that automagically happens if the setters are called after transformations are created